domain/infosync/info_test.go

```diff
@@ -70,7 +70,7 @@ func TestTopology(t *testing.T) {
 
 	topology, err := info.getTopologyFromEtcd(ctx)
 	require.NoError(t, err)
-	require.Equal(t, int64(1282967700000), topology.StartTimestamp)
```

Comment:
Why change the mock value? Is the previous too large to be meanningful?

Reply:
I'll change this to to use the original timestamp, but with `000` removed.

See also https://github.com/pingcap/tidb/pull/29790#issuecomment-979486931